examples/demo_test.py

```diff
@@ -0,0 +1,32 @@
+import sys
+
+sys.path.insert(0, "./examples/env")
+sys.path.insert(0, "./zoo")
+import baseline
+import gym
+import policies.keep_lane_agent
+
+from smarts.core.utils.episodes import episode_range
+from smarts.env.wrappers.episode_logger import EpisodeLogger
+from smarts.env.wrappers.record import AgentCameraRGBRender
+from smarts.env.wrappers.record_video import RecordVideo
+from smarts.zoo import registry
+
+if __name__ == "__main__":
```

Comment:
Perhaps this file can be removed in the final commit?

Reply:
Yes that is the intention.